mito-ai/style/ChatTaskpane.css

```diff
@@ -103,3 +107,83 @@
 .message-edit-buttons button:hover {
     background: #f0f0f0;
 }
+
+.chat-dropdown {
+    position: absolute;
+    width: 100%;
+    z-index: 9999;
```

Comment:
Spent some time working with `createPortal`; it's got potential, but it is a bit of a challenge to get it working properly. This is due to the somewhat complex nature of the sidebar, and needing the dropdown to be positioned relative to the textare. Things especially fall apart when the dropdown needs to be displayed above the textarea.

I'm going to suggest we put up with this forced scroll behavior for now, since users only scroll under very specific circumstances. When a user first enters the chat taskpane they should be fine, when the textare is at the bottom, users should also be fine. It's when the sidebar hasn't been filled yet, that the problem arises. 

Long term we could take a page from Cursor where the textarea starts on top, then is locked to the bottom after the first message.